### PR TITLE
Brice form

### PR DESCRIPTION
**Pull request by Elsass1**
I modified the following types

**type ToolCategory**
from _id: string to id: string

**type ToolCard**
from dailyRate: number  to dailyRate: number | string;

### CODE REVIEW

File: app/lib/types.ts [hunk 1]
```diff
@@ -30,7 +30,8 @@ export type ToolCard = {
 };
 
 export type ToolCategory = {
-  _id: string;
+  // _id: string;
```
Comment (Wyna-7) [inline app/lib/types.ts:33]: delete this commented out code

File: app/(no-nav)/form/page.tsx [hunk 2]
```diff
@@ -108,146 +120,171 @@ const Form = () => {
...
-              id='monthlyRate'
-              name='monthlyRate'
-              type='number'
-              placeholder='€'
-              value={input.monthlyRate || ''}
+              className='mb-4 border-b-2'
+              id='location'
+              name='location'
+              type='text'
+              placeholder='Where to pick up your tool?'
```
Comment (sevtechcodes) [inline app/(no-nav)/form/page.tsx:198]: Placeholder is  for currency. Is there any specific reason for the change?

File: app/(no-nav)/form/page.tsx [hunk 3]
```diff
@@ -53,15 +53,25 @@ const Form = () => {
...
         name === 'dailyRate' || name === 'weeklyRate' || name === 'monthlyRate'
-          ? parseInt(value)
+          ? value
+            ? parseInt(value)
+            : 0
           : value,
     }));
   };
 
+  // const handleSelectChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
```
Comment (Wyna-7) [inline app/(no-nav)/form/page.tsx:68]: delete commented out code if not needed anymore, or is this for future functionality implementation?

File: app/(no-nav)/form/page.tsx [hunk 4]
```diff
@@ -108,146 +120,171 @@ const Form = () => {
...
-        </div>
-        <form
-          onSubmit={handleSubmit}
-          className='flex flex-col bg-white shadow-md rounded px-8 pt-6 pb-8 mb-4'
-        >
-          <label htmlFor='name' className='mb-1'>
+    <>
+      <header className='flex justify-start pt-4 pl-5  border-grey h-20 shadow-md mb-5 bg-slate-200 '>
+        <Link href='/rented'>
+          {/* <ChevronLeftIcon className='h-15 w-11  border ml-5 pt-5'/> */}
```
Comment (Wyna-7) [inline app/(no-nav)/form/page.tsx:149]: if we won't need the chevron, we ca delete this line

**Review by Wyna-7 — CHANGES_REQUESTED** [review]

**Review by sevtechcodes — APPROVED** [review]

**Review by Wyna-7 — APPROVED** [review]

**Pull request merged**